.github/ISSUE_TEMPLATE.md

```diff
@@ -8,7 +8,9 @@
 
 ### Replicate:
 
-1.  2.  3.
+1.  <!-- Step 1 to Replicate the bug -->
```

Comment:
This isn't really fixed. People will have to know to do this whenever they do a list in markdown.

I know it's tangential, but in some future social experiment, I'd be interested to see if people who don't know HTML and are filling in the template will get confused about where they're supposed to put their text...